mlflow/gateway/cli.py

```diff
@@ -38,5 +38,14 @@ def commands():
     default=2,
     help="The number of workers.",
 )
-def start(config_path: str, host: str, port: str, workers: int):
-    run_app(config_path=config_path, host=host, port=port, workers=workers)
+@click.option(
+    "--dev",
+    is_flag=True,
+    default=False,
+    help=(
+        "If True, run the service with options suitable for development. "
+        "Should not be used in production."
+    ),
```

Comment:
Can we describe what some of these arguments are? If users aren't supposed to run this in prod, should we just make this an env var that we can set? I'd prefer that :)

Reply:
@dbczumar Found a better approach: https://github.com/mlflow/mlflow/pull/8713